OpenRA.Mods.Common/Activities/UnloadCargo.cs

```diff
@@ -124,15 +124,16 @@ public override Activity Tick(Actor self)
 
 			if (!unloadAll || !cargo.CanUnload())
 			{
-				Cancel(self, true);
```

Comment:
What does removing this cancel achieve?

Reply:
The same effect can be achieved by returning `true` instead. So this is simply to make it consistent with the other changes.